primitives/avn-common/src/lib.rs

```diff
@@ -173,13 +173,20 @@ pub fn recover_public_key_from_ecdsa_signature(
     }
 }
 
-pub fn hash_with_ethereum_prefix(hex_message: String) -> Result<[u8; 32], ECDSAVerificationError> {
+pub fn hash_with_ethereum_prefix(hex_message: &String) -> Result<[u8; 32], ECDSAVerificationError> {
```

Comment:
If we are centralising I think we should use the version that accepts bytes (the one in avn-service) and replace this function with that. The conversion `toString()` might affect the bytes so its safer to work with bytes.

Reply:
well both convert string to bytes, trim `0x` prefix.
The one in avn-service does it outside the function, the one in avn-common does it bundled in a fucntion.
So it makes more sense to keep this version.
That is till we replace it all together with the one provided by ethers-rs implementation